autotest/common.py

```diff
@@ -25,7 +26,15 @@ def mf6_dev_no_final_check(model_ws, fname):
 
 
 def dll_loc():
-    return os.path.join("..", "bin", "libmf6")
+    cplatform = platform.platform().lower()
+    if 'window' in cplatform:
+        return os.path.join("..", "bin", "libmf6.dll")
+    elif ('mac' in cplatform) and ('arm' in cplatform):
+        return os.path.join("..", "bin", "mac_arm", "libmf6.dylib")
+    elif ('mac' in cplatform) and ('arm' not in cplatform):
+        return os.path.join("..", "bin", "mac_x86_64", "libmf6.dylib")
+    else:
+        return os.path.join("..", "bin", "libmf6.so")

```

Comment:
Thanks for figuring out the arm vs. intel platform issue on macos. I'll pull this into the repo and then will be moving this code from `common.dll_loc()` to `ModflowApiAg.run_model()`